src/dart/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
```

Comment:
are you open to use the main license for this repo?
https://github.com/sign-language-processing/pose/blob/master/LICENSE.txt

Reply:
I could replace/remove the license.